mlserver/settings.py

```diff
@@ -146,6 +146,11 @@ class Config:
     kafka_topic_input: str = "mlserver-input"
     kafka_topic_output: str = "mlserver-output"
 
+    # Custom server settings
+    custom_rest_server_settings: Optional[dict] = None
```

Comment:
Should we prefix these fields with an underscore (i.e. as `_custom_rest_server_settings`)? This would mainly be to highlight that these settings are provided "as-is" to users and not supported.

Happy to hear other suggestions though! Perhaps we could also add a warning message when they are used?

Reply:
I think both are good ideas!! I've added the prefix and a warning message.